src/renderers/webgl/WebGLBackground.js

```diff
@@ -67,6 +75,12 @@ function WebGLBackground( renderer, cubemaps, cubeuvmaps, state, objects, alpha,
 
 		}
 
+	}
+
+	function addToRenderList( renderList, scene ) {
```

Comment:
Actually I like the distinction between `render()` and `addToRenderList()`. It makes the code more clear. Whereas `render()` can actually write data to the framebuffer, `addToRenderList()` only manipulates the render list without actually rendering something.